minpy/array.py

```diff
@@ -558,6 +585,11 @@ def T(self):
         return Value._ns.transpose(self)
     # pylint: enable= invalid-name
 
+    @property
+    def size(self):
+        """ Get number of elements in the array """
+        return Value._ns.prod(self.shape)
```

Comment:
Use the `mxnet.ndarray.NDArray.size` function for mxnet type. Don't use `prod` function since it will incur possible GPU computation.